cdap-common/src/main/java/co/cask/cdap/common/metrics/MetricsTags.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.metrics;
+
+import co.cask.cdap.proto.Id;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * Provides ways to obtain the context for certain metrics.
+ */
+public final class MetricsTags {
```

Comment:
no i meant, we don't have to remove MetricContexts, the name is misleading, we are just returning the map of tags there and not a context string, so we can just rename that to be MetricTags right? the functions are similar to MetricTags as @anwar6953  mentioned. let me know if i am missing something.